core/src/main/java/io/varhttp/VarServlet.java

```diff
@@ -95,10 +105,24 @@ public void handle(HttpServletRequest request, HttpServletResponse response) {
 		}
 	}
 
+
+
 	public void configure(Consumer<VarConfiguration> configuration) {
-		VarConfiguration varConfiguration = new VarConfiguration(this, controllerMapper, baseConfigurationContext, parameterHandler);
+		VarConfiguration varConfiguration = new VarConfiguration(this, controllerMapper, baseConfigurationContext, parameterHandler, registeredWebSockets, null);
 		configuration.accept(varConfiguration);
 		baseConfigurationContext.applyMappings();
 		varConfiguration.applyMappings();
 	}
+
+	@Override
+	public void destroy() {
+		super.destroy();
```

Comment:
should the websockets not be closed before super.destroy()?

Reply:
Please provide more context to your comment. Why does the order of destruction matter?